hadoop-ozone/dev-support/checks/rat.sh

```diff
@@ -16,7 +16,10 @@
 
 mkdir -p target
 rm target/rat-aggregated.txt
-mvn -fn org.apache.rat:apache-rat-plugin:0.13:check -am -pl :hadoop-ozone-dist -Phdds
+cd hadoop-hdds
+mvn -B -fn org.apache.rat:apache-rat-plugin:0.13:check
+cd ../hadoop-ozone
```

Comment:
shellcheck:1: warning: Use 'cd ... || exit' or 'cd ... || return' in case cd fails. [SC2164]